framework/contracts/native/ibc-client/src/ibc.rs

```diff
@@ -88,14 +88,17 @@ pub fn receive_action_callback(
                     .add_attribute("chain", host_chain.to_string()),
             )
         }
-        IbcClientCallback::UserRemoteAction(callback_info) => {
-            // Here we transfer the callback back to the module that requested it
-            let callback = IbcResponseMsg {
+        IbcClientCallback::ModuleRemoteAction {
+            callback_info,
+            sender_module,
+        } => {
+            let callback = IbcCallbackMsg {
```

Comment:
We have access to the initator message in this function. Would it not be nice to have it included in the callback as well? 

`callback.initiator_msg`

Reply:
So we'll have a callback message as well as an initiator_msg in there, that's maybe too much ?